apps/filebrowser/src/filebrowser/templates/listdir_components.mako

```diff
@@ -1449,6 +1451,18 @@ else:
           $('#newRepFactorInput').val(self.selectedFile().stats.replication);
         });
       };
+      
+      self.allowCopyMoveTo = function (destination) {
+        const removeLastSlash = function (path) {
+          if (path.charAt(path.length - 1) === '/') {
+            return path.slice(0, -1);
+          }
+          return path;
+        };
+
+        const source = self.currentPath();
```

Comment:
Is the path always case sensitive?

Reply:
Very good question. For S3, HDFS and ABFS that is the case at least. @Harshg999 do you know if we are supporting any case insensitive file systems?

Reply:
Apart from the scheme(like s3a:// or S3A://, abfs:// or ABFS:// etc.), everything should be case-sensitive in the filesystem path.